### ISSUE TITLE
Import/Export Configs for plugins

### ISSUE THREAD

**Issue by Bl69aze** [runelite/runelite#5248]
I would like to suggest a feature a friend brought to my mind that could be useful.

The ability to Import or Export your plugin configs, For example i made a perfect route in Rogues den using TILE MARKERS, where to click for the best way to go.

Friend asked me to show him all the tiles, and thats when i had an idea that it would be cool if i could somehow share my custom marked tiles to someone else :)

TL;DR - Sharing configs with friends easily

**Comment by deathbeam** [comment]
duplicate
